packages/itwin/property-grid/src/components/PropertyGridContent.tsx

```diff
@@ -93,13 +94,36 @@ export function PropertyGridContent({
 
   const [filterText, setFilterText] = useState<string>("");
   const { showNullValues } = useNullValueSettingContext();
+  const { onFeatureUsed } = useTelemetryContext();
   const filterer = useFilterer({ showNullValues, filterText });
 
+  const filterTextRef = useLatest(filterText);
+  const showNullValuesRef = useLatest(showNullValues);
+
   const [{ width, height }, setSize] = useState({ width: 0, height: 0 });
   const handleResize = useCallback((w: number, h: number) => {
     setSize({ width: w, height: h });
   }, []);
 
+  const reportFilterFeature = useCallback(() => {
```

Comment:
I think this should be done in `onSearchTextChange` callback.

Reply:
Moved to `onSearchTextChange`.